### PR TITLE
zipcloak: add Spanish translation

### PR DESCRIPTION
**Pull request by GustavoHiguera**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages.es/linux/zipcloak.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# zipcloak
+
+> Encripta el contenido de un archivo zip.
+> Más información en: <https://manned.org/zipcloak>.
```
Comment (kbdharun) [inline pages.es/linux/zipcloak.md:4]:
```suggestion
> Más información:: <https://manned.org/zipcloak>.
```

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

File: pages.es/linux/zipcloak.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+`zipcloak {{ruta/al/archivo.zip}}`
+
+- [d]esencripta el contenido de un archivo zip:
+
+`zipcloak -d {{ruta/al/archivo.zip}}`
+
+- Genera [O] un nuevo archivo zip con el contenido encriptado:
+
+`zipcloak {{ruta/al/archivo.zip}} -O {{ruta/al/archivo.zip}}`
```
Comment (navarroaxel) [inline pages.es/linux/zipcloak.md:16]: This is a better translation
 ```suggestion
`zipcloak {{ruta/al/archivo.zip}} -O {{ruta/al/encriptado.zip}}`
```
`archivo_encriptado` is another option too.

File: pages.es/linux/zipcloak.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+- Encripta el contenido de un archivo zip:
+
+`zipcloak {{ruta/al/archivo.zip}}`
+
+- [d]esencripta el contenido de un archivo zip:
+
+`zipcloak -d {{ruta/al/archivo.zip}}`
+
+- Genera [O] un nuevo archivo zip con el contenido encriptado:
```
Comment (navarroaxel) [inline pages.es/linux/zipcloak.md:14]: I think we can use the highlight here
```suggestion
- Genera un nuev[O] archivo zip con el contenido encriptado:
```

File: pages.es/linux/zipcloak.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+
+`zipcloak {{ruta/al/archivo_encriptado.zip}}`
+
+- [d]esencripta el contenido de un archivo zip:
+
+`zipcloak -d {{ruta/al/archivo_encriptado.zip}}`
+
+- Genera un nuev[O] archivo zip con el contenido encriptado:
+
+`zipcloak {{ruta/al/archivo_encriptado.zip}} -O {{ruta/al/archivo_encriptado.zip}}`
```
Comment (navarroaxel) [inline pages.es/linux/zipcloak.md:16]: The first file isn't encrypted
```suggestion
`zipcloak {{ruta/al/archivo.zip}} -O {{ruta/al/archivo_encriptado.zip}}`
```

File: pages.es/linux/zipcloak.md [hunk 5]
```diff
@@ -0,0 +1,16 @@
+# zipcloak
+
+> Encripta el contenido de un archivo zip.
+> Más información: <https://manned.org/zipcloak>.
+
+- Encripta el contenido de un archivo zip:
+
+`zipcloak {{ruta/al/archivo_encriptado.zip}}`
```
Comment (navarroaxel) [inline pages.es/linux/zipcloak.md:8]: This file isn't encrypted, the command will encrypt it
```suggestion
`zipcloak {{ruta/al/archivo.zip}}`
```

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**